demos/sample_azure_iot_adu/sample_azure_iot_pnp_simulated_data.c

```diff
@@ -235,6 +235,62 @@ static uint32_t prvGetNewMaxTemp( double xUpdatedTemperature,
 }
 /*-----------------------------------------------------------*/
 
+
+static bool prvIsUpdateAlreadyInstalled( const AzureIoTADUUpdateRequest_t * pxAduOtaUpdateRequest )
```

Comment:
Nit: I think this might need to have different file/sample since ADU is not free/or available on Basic SKUs.

Reply:
that's an interesting point. For now, the sample is in a separate `demos/sample_azure_iot_adu` directory.